public/docs/ts/latest/guide/reactive-forms.jade

```diff
@@ -0,0 +1,903 @@
+  bind them to _data model_ properties in the component, using directives 
+  like `[(ngModel)]`.
+
+  You don't create Angular form control objects. Angular directives 
+  create them for you, using the information in your data bindings. 
+  You don't push and pull data values. Angular handles that for you with `ngModel`.
+  Angular updates the mutable _data model_ with user changes as they happen.
+
+  When writing in a reactive style, you determine the timing and flow of data between 
+  the data model and the screen. You rarely, if ever, use two-way data binding. 
+  For this reason, the ngModel directive is not part of the ReactiveFormsModule.
+  
+  These differences reflect two architectural paradigms, 
+  with their own strengths and weaknesses,
+  and you are free to choose between them.
+
+  The balance of this _reactive forms_ guide assumes the _reactive_ paradigm and 
+  concentrates exclusively on the reactive forms techniques. For 
+  information on _template driven forms_, see the [Template Guide](forms.html).
+
+  You'll learn about reactive forms by building one from scratch.
+
+  In the next section, you'll prepare your project for reactive forms.
+  Then you'll [learn about the Angular form classes](#essentials) and how to use them in a reactive form.
+
+.l-main-section
+a#setup
+:marked
+  ## Setup
+
+  Follow the steps in the [_Setup_ guide](../setup.html "Setup guide") 
+  for creating a new project folder (perhaps called `reactive-forms`)
+  based on the _QuickStart seed_.
+
+.l-main-section
+a#data-model
+:marked
+  ## Create a data model
+  The focus of this guide is a reactive forms component that edits a hero.
+  You'll need a `hero` class and some hero data.
+  Create a new `data-model.ts` file in the `app` directory and copy the content below into it.
+
++makeExample('reactive-forms/ts/app/data-model.ts', '','app/data-model.ts')(format=".")
+
+:marked
+  The file exports two classes and two constants. The `Address` 
+  and `Hero` classes define the application _data model_. 
+  The `heroes` and `states` constants supply the test data. 
+
+.l-main-section
+a#create-component
+:marked
+  ## Create a _reactive forms_ component
+  Make a new file called 
+  `hero-detail.component.ts` in the `app` directory and import these symbols:
+
++makeExample('reactive-forms/ts/app/hero-detail-1.component.ts', 'imports','app/hero-detail.component.ts')(format=".")
+
+:marked
+  Now enter the `@Component` decorator that specifies the `HeroDetailComponent` metadata:
+
++makeExample('reactive-forms/ts/app/hero-detail.component.ts', 'metadata','app/hero-detail.component.ts (excerpt)')(format=".")
+
+:marked
+  The `moduleId: module.id` lets you use 
+  [component-relative paths](./cookbook/component-relative-paths.html) in file URLs 
+  such as when specifying the `templateUrl`.
+
+  Next, create an exported `HeroDetailComponent` class with a `FormControl`. 
```

Comment:
@kara What would that look like. You can see the progression now.  How would one do a control all by itself in the absence of the form envelope? Does that mean the `<form>` element is not in the html for that example.  TBH, we are confused about both the use case and how to code it. A code snippet or plunker would help.